content/docs/tutorials/cloudfx/_index.md

```diff
@@ -1,29 +1,29 @@
 ---
-title: Pulumi Cloud Framework (Preview)
+title: Cloud Framework Preview
 meta_desc: The Cloud framework for Pulumi lets you program infrastructure and application logic using high-level
            cloud-agnostic building blocks.
 aliases: ["/docs/quickstart/cloudfx/"]
 ---
 
 <img src="/images/docs/quickstart/cloudfx-purple.png" align="right">
 
-The Cloud framework for Pulumi lets you program infrastructure and application logic, side by side, using simple, high-level, cloud-agnostic building blocks.
+The Cloud Framework for Pulumi lets you program infrastructure and application logic, side by side, using simple, high-level, cloud-agnostic building blocks.
 
-The Cloud framework must be configured with credentials to deploy and update resources in the target cloud platform.
+The Cloud Framework must be configured with credentials to deploy and update resources in the target cloud adoption platform.
 
-See the [full API documentation]({{< relref "/docs/reference/pkg/nodejs/pulumi/cloud" >}}) for complete details of the available Cloud framework APIs.
+See the [full API documentation]({{< relref "/docs/reference/pkg/nodejs/pulumi/cloud" >}}) for complete details of the available Cloud Framework [APIs](https://www.pulumi.com/docs/reference/pkg/).
 
 For AWS-specific use cases, see also the [awsx]({{< relref "/docs/reference/pkg/nodejs/pulumi/awsx" >}}) library which provides higher-level libraries for working with many AWS services.
 
 ## Getting Started
 
-The easiest way to start with the Cloud Framework is to follow one of the tutorials:
+The easiest way to start with the Cloud Framework is to follow one of the tutorials for adoption:
```

Comment:
This feels a bit off to me. I think we should remove the "for adoption" end of this sentence.

Reply:
```suggestion
The easiest way to start with the Cloud Framework is to follow one of the tutorials:
```